signup/app/signup/signup.component.ts

```diff
@@ -13,20 +12,17 @@ export class SignupComponent implements OnInit {
     email: string;
     password: string;
 
-    constructor(private routerExtensions: RouterExtensions) {
+    constructor() {
```

Comment:
I thought that we want empty methods and constructors? 
To show the people where to put their stuff.